streams/voxel_stream.cpp

```diff
@@ -70,9 +70,14 @@ bool VoxelStream::has_script() const {
 }
 
 void VoxelStream::_bind_methods() {
-	// TODO Make these proper virtual, it confuses C# bindings
 	// Note: C++ inheriting classes don't need to re-bind these, because they are bindings that call the actual virtual methods
-	ClassDB::bind_method(D_METHOD("emerge_block", "out_buffer", "origin_in_voxels", "lod"), &VoxelStream::_emerge_block);
-	ClassDB::bind_method(D_METHOD("immerge_block", "buffer", "origin_in_voxels", "lod"), &VoxelStream::_immerge_block);
-	ClassDB::bind_method(D_METHOD("get_used_channels_mask"), &VoxelStream::_get_used_channels_mask);
+	BIND_VMETHOD(MethodInfo("emerge_block",
```

Comment:
Same here, if no method pointers are specified, none of the voxel streams will be usable from scripts unless you inherit them with a script, which is not always the case